src/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,224 @@
+                  this._handleTabClick(TabName.OVERVIEW);
+                }}
+              >
+                  Overview
+              </a>
+            </li>
+            <li
+              className={`movie-nav__item ${this._getActiveTabClass(
+                  TabName.DETAILS
+              )}`}
+            >
+              <a
+                className="movie-nav__link"
+                onClick={() => {
+                  this._handleTabClick(TabName.DETAILS);
+                }}
+              >
+                 Details
+              </a>
+            </li>
+            <li
+              className={`movie-nav__item ${this._getActiveTabClass(
+                  TabName.REVIEWS
+              )}`}
+            >
+              <a
+                className="movie-nav__link"
+                onClick={() => {
+                  this._handleTabClick(TabName.REVIEWS);
+                }}
+              >
+                  Reviews
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+        {activeTab === TabName.OVERVIEW && (
+          <React.Fragment>
+            <div className="movie-rating">
+              <div className="movie-rating__score">{film.ratingScore}</div>
+              <p className="movie-rating__meta">
+                <span className="movie-rating__level">
+                  {this._getRatingLevel(film.ratingScore)}
+                </span>
+                <span className="movie-rating__count">
+                  {film.ratingCount} ratings
+                </span>
+              </p>
+            </div>
+
+            <div className="movie-card__text">
+              <p>{film.description}</p>
+
+              <p className="movie-card__director">
+                <strong>Director: {film.director}</strong>
+              </p>
+
+              <p className="movie-card__starring">
+                <strong>
+                    Starring: {`${film.starring.join(`, `)} and other`}
+                </strong>
+              </p>
+            </div>
+          </React.Fragment>
+        )}
+
+        {activeTab === TabName.DETAILS && (
+          <React.Fragment>
+            <div className="movie-card__text movie-card__row">
+              <div className="movie-card__text-col">
+                <p className="movie-card__details-item">
+                  <strong className="movie-card__details-name">Director</strong>
+                  <span className="movie-card__details-value">
+                    {film.director}
+                  </span>
+                </p>
+                <p className="movie-card__details-item">
+                  <strong className="movie-card__details-name">Starring</strong>
+                  <span className="movie-card__details-value">
+                    {film.starring.map((actor, i) => (
+                      <React.Fragment key={i}>
+                        {`${actor}, `} <br/>
+                      </React.Fragment>))}
+                  </span>
```

Comment:
```suggestion
                  <span className="movie-card__details-value">
                    {film.starring.map((actor, i) => (
                      <React.Fragment key={actor}>
                        {actor}
                        {i < film.starring.length - 1 && <React.Fragment>,<br/></React.Fragment> }
                      </React.Fragment>))}
                  </span>
```
чтобы не было запятой в конце